test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -123,6 +123,9 @@ def test_nvapex(pytorch_training, ec2_connection, gpu_only):
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
 @pytest.mark.skipif(PT_EC2_GPU_INSTANCE_TYPE == ["g3.4xlarge"], reason="Skipping AMP DDP test on single gpu instance")
 def test_pytorch_amp(pytorch_training, ec2_connection, gpu_only):
+    from packaging import Version
```

Comment:
When the CI tests on this PR are completed successfully, please move this import line to the top of the pytest script along with all the other standard module imports (i.e., below `import os` for standard module `os`, and above `import pytest` for 3P module `pytest`).

Reply:
Also, this should be `from packaging.version import Version`.